components/logins/src/db.rs

```diff
@@ -577,12 +609,43 @@ impl LoginDb {
                 Ok(_) => log::info!("Imported {} (new GUID {}) successfully.", old_guid, guid),
                 Err(e) => {
                     log::warn!("Could not import {} ({}).", old_guid, e);
-                    num_failed += 1;
+                    insert_errors.push(format!("Could not import {} ({}).", old_guid, e));
+                    num_failed_insert += 1;
                 }
             };
         }
         tx.commit()?;
-        Ok(num_failed)
+        // TODO: handle potential negative durations and counts
+        let fixup_phase_duration = fixup_phase_end - import_start;
+        let num_post_fixup = import_start_total_logins - num_failed_fixup;
+
+        let insert_phase_end = util::system_time_ms_i64(SystemTime::now());
+        let insert_phase_duration = insert_phase_end - insert_phase_start;
```

Comment:
Please log these values in addition to returning them, we'll probably find that easier for local testing.

Reply:
Would a `log::info!` of the `MigrationMetrics` I'm returning suffice? Also, why is that easier for local testing? I might be missing something here.

Reply:
> Would a log::info! of the MigrationMetrics I'm returning suffice? 

Sounds good. I meant it could be useful in a very general "we could run a Fenix built with this code and watch the output of `adb logcat` to see what is happening" kind of way, not in a useful-for-structured-testing kind of way.

Reply:
Ah, gotcha--will do.